scripts/deploy.py

```diff
@@ -284,12 +308,12 @@ def _execute_deployment():
     # error logs.
     if (APP_NAME == APP_NAME_OPPIATESTSERVER or 'migration' in APP_NAME) and (
             _get_served_version() == _get_current_release_version()):
+        common.open_new_tab_in_browser_if_possible(
```

Comment:
I don't recollect this happening when I released this time.... Any situations when it isn't "possible"?

Reply:
That typically happens when the script can't find where the command to open the browser is (maybe the executable is in a different place from normal).

Reply:
Actually, I just realized that if this doesn't work for you, we should figure out why. See https://github.com/oppia/oppia/blob/develop/scripts/common.py#L50 -- do you know why that doesn't work on your machine?

Reply:
I had to mess around with my $PATH a little bit (basically all 3 browsers didn't have commands executable from the command-line). Now the script does open tabs :)

Reply:
Looks like by default on mac, the executables aren't in the $PATH...

Reply:
Hm. I think in a separate PR I might make the instructions clearer to say "please open these links manually" if we can't open them by the script. I'll look into that!

Reply:
Actually, I've done that in this PR. PTAL!